src/wymeditor/editor/base.js

```diff
@@ -2186,18 +2080,17 @@ WYMeditor.editor.prototype.insert = function (html) {
     }
 };
 
-WYMeditor.editor.prototype.wrap = function (left, right) {
-    var wym = this;
-
-    wym.insert(
-        left + wym._iframe.contentWindow.getSelection().toString() + right
-    );
-};
```

Comment:
Bam! No `wrap`. I had to remove the "more inline elements" example, which was kind of broken anyway. Changelog entry added.